### ISSUE TITLE
[Question] why is there no XNAS (NASDAQ) calendar?

### ISSUE THREAD

**Issue by xareelee** [gerrymanoim/exchange_calendars#302]
I'm using polygon.io API which has market data for the following US exchanges: 

- XNYS (New York Stock Exchange)
- XNAS (NASDAQ)
- XASE (NYSE American, LLC)
- ARCX (NYSE Arca, Inc.)
- BATS (Cboe BZX)

I found that [exchange_calendars](https://github.com/gerrymanoim/exchange_calendars) does not provide XNAS calendar, why?

Do those exchanges always share the same calendar?

**Comment by maread99** [comment]
To get a list of all supported calendar names:
```python
import exchange_calendars as xcals
xcals.get_calendar_names()
```
You'll see these include NASDAQ, CBOT, COMEX etc...

Some of these names are aliases for other calendar codes. For example:
```python
>>> cal = xcals.get_calendar("NASDAQ")
>>> cal.name
"XNYS"
```

**Comment by maread99** [comment]
closed by #318
